bitnami/redis/templates/scripts-configmap.yaml

```diff
@@ -140,6 +145,14 @@ data:
     . /opt/bitnami/scripts/libvalidations.sh
     . /opt/bitnami/scripts/libfile.sh
 
+    myip=$(hostname -i)
+    
+    #If two IPs, use 2nd one (IPv4 in dualstack environment)
```

Comment:
Is it 100% certain for the IPv4 address to be the second address in dualstack environments?

Reply:
As far as I can there's no clear answer to that. "hostname -i" just iterates through h_addr_list in gethostbyname(), and that's documented as "An array of pointers to network addresses for the host (in network byte order), terminated by a null pointer."  (see: https://man7.org/linux/man-pages/man3/gethostbyname.3.html )

Since this is going to dictate how the other comments go, what's the preferred approach here?  As I see it we could either:
- Just assume that the 2nd address is going to be ipv4, in which case we just want to make the cleanup changes
or
- Switch to something a little more heavyweight like an awk script or a bash regex to parse the input line for something that looks like an ipv4 regex?

Reply:
Switched to awk to detect ipv4.  Had to use a slightly different syntax for the sentinel regex since that container is using mawk instead of gnu awk like the redis container does.  mawk doesn't support the {3} syntax in regexes